### PR TITLE
LPS-129699 Remove Liferay.Util.selectEntityHandler usage from roles

### PR DESCRIPTION
**Pull request by kresimir-coko**
This PR removes usages of `Liferay.Util.selectEntityHandler` inside `roles-admin-web`. I've manually tested the changes to make sure modals work as they did before.

## 🧪 Testing
1. Open the Control Panel and go to Users and Organizations
2. Create a new user
3. Click on Roles in the left navigation
4. `Select` Regular roles, and `Select` Site roles

### CODE REVIEW

**Comment by kresimir-coko** [comment]
@markocikos helped me discovered that [users-admin-web/roles.jsp](https://github.com/liferay/liferay-portal/blob/master/modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp#L545-L642) is the one that handles the modal in question, so I'll migrate that one and send it.

**Comment by kresimir-coko** [comment]
Hey @markocikos can you give this a look now? 
I've replaced the `Liferay.Util.selectEntity` with `Liferay.Util.openSelectionModal` in the `users-admin-web/roles.jsp` and added the removal of the other `selectEntityHandler` from `roles-web`.

I've tested to make sure it still works. LMK if I missed something.

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp [hunk 1]
```diff
@@ -588,57 +588,49 @@ currentURLObj.setParameter("historyKey", liferayPortletResponse.getNamespace() +
...
+				).setWindowState(
+					LiferayWindowState.POP_UP
+				).build();
+				%>
 
-							<%
-							String siteRoleEventName = liferayPortletResponse.getNamespace() + "selectSiteRole";
-							%>
+				var selectSiteRoleLink = document.querySelector(
+					'#<portlet:namespace />selectSiteRoleLink'
```
Comment (markocikos) [inline modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp:610]: Minor SF: we can use `getElementById`.

Also, we can use `const`.

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp [hunk 2]
```diff
@@ -588,57 +588,49 @@ currentURLObj.setParameter("historyKey", liferayPortletResponse.getNamespace() +
...
-								event.searchcontainername,
-								event.groupdescriptivename,
-								event.groupid,
-								event.iconcssclass
-							);
-						}
-					);
-				});
+							url: '<%= selectSiteRoleURL.toString() %>',
+							onSelect: function (selectedItem) {
```
Comment (markocikos) [inline modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp:621]: Minor SF: sort, `url` > `onSelect`

**Review by markocikos — COMMENTED** [review]
@kresimir-coko Looks not horrible at all! Some minor nits inline.

**Comment by jonmak08** [comment]
LGTM otherwise the small SF changes @markocikos commented on.

**Comment by kresimir-coko** [comment]
Manually forwarded to https://github.com/pei-jung/liferay-portal/pull/823

**Pull request closed without merging**